cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -54,6 +54,13 @@ def _decorator(request, *args, **kwargs):
     return _check_feature_enabled
 
 
+def is_entrance_exams_enabled():
+    """
+    Returns a boolean indicating entrance exam feature is enable or not.
+    """
+    return settings.FEATURES.get('ENTRANCE_EXAMS', True)
```

Comment:
@asadiqbal08 good catch default for this feature flag should be `False`. Is there anything else you think needs to be changed?